example/portmonitor/depth_image/DepthImage.cpp

```diff
@@ -0,0 +1,93 @@
+// -*- mode:C++; tab-width:4; c-basic-offset:4; indent-tabs-mode:nil -*-
+
+/*
+ * Copyright (C) 2016 iCub Facility
+ * Authors: Alberto Cardellino
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ */
+
+#include <stdio.h>
+#include <math.h>
+#include <algorithm>
+#include <yarp/sig/Image.h>
+#include <yarp/os/SharedLibraryClass.h>
+#include "DepthImage.h"
+
+using namespace yarp::os;
+using namespace yarp::sig;
+
+YARP_DEFINE_SHARED_SUBCLASS(MonitorObject_there, DepthImageConverter, MonitorObject);
+
+bool DepthImageConverter::create(const yarp::os::Property& options)
+{
+    min = 0.2;
+    max = 10.0;
+    inMatrix = NULL;
+    outMatrix = NULL;
+    printf("created!\n");
```

Comment:
please remove this and the following `printf`, if these are important information use `yDebug` and similar methods.